site/_components/community-quotes.njk

```diff
@@ -0,0 +1,48 @@
+<!--{#
+  Renders a flexible grid of quote figures.
+
+  @param {object} content - An object with the following attributes.
+  @param {string} content.title - Section heading
+  @param {string} content.subtitle (optional) - Section subtitle, displayed only when a title is also included.
+
+  @param {object[]} quotes - An array of quote objects with the following attributes.
+  @param {string} quotes[].copy: The body content of the quote.
+  @param {object} quotes[].author - An object with the following attributes.
+  @param {string} quotes[].author.name - The name of the quote author.
+  @param {string} quotes[].author.role - The role of the quote author.
+  @param {object} quotes[].image - An object with the following attributes.
+  @param {string} quotes[].image.src - Image path.
+  @param {string} quotes[].image.alt - Image description.
+  @param {string[]} quotes[].decorators - array of decorative image sources.
+#}-->
+{% macro communityQuotes(content, quotes) %}
+<section class="community-quotes">
+  {% if content.title is defined %}
+  <header class="community-quotes__header">
+    <h1 class="type--h1 community-quotes__title">{{content.title}}</h1>
+    {% if content.subtitle is defined %}
+    <p class="type--large community-quotes__subtitle">{{content.subtitle}}</p>
+    {% endif %}
+  </header>
+  {% endif %}
+  <div class="community-quotes__content">
+    {% for quote in quotes %}
+    <figure class="community-quotes__item">
+      {% for decorator in quote.decorators %}
+      <img class="community-quotes__decorator" src="{{decorator}}" alt="" aria-hidden="true" />
+      {% endfor %}
+      <img class="community-quotes__image" src="{{quote.image.src}}" alt="{{quote.image.alt}}" />
+      <figcaption class="community-quotes__caption">
+        <blockquote class="community-quotes__block">
+          <p class="type--h4">{{quote.content.copy}}</p>
+          <cite class="community-quotes__citation type--caption">
+            <span class="community-quotes__author">{{quote.content.author.name}}</span>
+            {{quote.content.author.role}}
+          </cite>
```

Comment:
Reading through HTMl tag semantics, I believe this should be wrapped in a `footer` tag to signify that this is the author of the section (quote). See [HTML5 Doctor's `cite` article](http://html5doctor.com/cite-and-blockquote-reloaded/)